modules/bench/bin/auctionmark-compare.clj

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bb
+
+(require '[babashka.cli :as cli]
+         '[babashka.process :as p])
+
+(def debug true)
+
+(def cli-opts {:coerce {:load-phase :boolean :duration :string}})
+(def opts (cli/parse-opts *command-line-args* cli-opts))
+
+(def c1-auctionmark-file "/tmp/auctionmark-c1-run.edn")
+(def c2-auctionmark-file "/tmp/auctionmark-c2-run.edn")
+(def jvm-args ["-Xmx2g", "-Xms2g", "-XX:MaxDirectMemorySize=3g", "-XX:MaxMetaspaceSize=1g"])
+
+(when debug (prn opts))
```

Comment:
is this the idiomatic way to check the logging level on bb? I've not used it a lot, but I'd guess they'd have something similar to c.t.logging?

Reply:
I can add normal logging. I just thought as this is some hacky script it doesn't matter much.